traptor/traptor.py

```diff
@@ -138,6 +139,11 @@ def _setup_birdy(self):
                                         self.apikeys['ACCESS_TOKEN_SECRET']
                                         )
 
+    @retry(wait=wait_exponential(multiplier=1, max=10),
+           stop=stop_after_attempt(3),
+           reraise=True,
+           retry=retry_if_exception_type(KafkaUnavailableError)
```

Comment:
This only handles the `KafkaUnaviailableError` scenario right?  What about connect timeout, network drops, or other errors?  What I did in vkollect is creating a new class called TenaciousProducer that handles all the kafka errors, trying the producer if it fails, and re-creating it as a last try.

What you have here is fine but I think we need a more all encompassing retry solution for kafka.